From fd68c45210d4543618a237389db3bfb3ffcd0400 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@altera.com>
Date: Mon, 13 Jan 2014 10:18:17 -0600
Subject: [PATCH 172/248] FogBugz #177545: Remove the need to force the USB IP
 into Host mode

Users must correctly use a USB OTG A-mini to A Male cable adapter and the USB
IP will correctly transistion into HOST mode.

Re-align the DWC2 USB driver with kernel.org.

[Original patch taken from
git://git.rocketboards.org/linux-socfpga.git socfpga-3.14]

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Dinh Nguyen <dinguyen@altera.com>
---
 drivers/usb/dwc2/core.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/dwc2/core.c b/drivers/usb/dwc2/core.c
index 5bee748..1d12988 100644
--- a/drivers/usb/dwc2/core.c
+++ b/drivers/usb/dwc2/core.c
@@ -2624,12 +2624,10 @@ int dwc2_get_hwparams(struct dwc2_hsotg *hsotg)
 
 	hptxfsiz = readl(hsotg->regs + HPTXFSIZ);
 	dev_dbg(hsotg->dev, "hptxfsiz=%08x\n", hptxfsiz);
-/*
 	gusbcfg = readl(hsotg->regs + GUSBCFG);
 	gusbcfg &= ~GUSBCFG_FORCEHOSTMODE;
 	writel(gusbcfg, hsotg->regs + GUSBCFG);
 	usleep_range(100000, 150000);
-*/
 
 	/* hwcfg2 */
 	hw->op_mode = (hwcfg2 & GHWCFG2_OP_MODE_MASK) >>
-- 
1.9.1

